test/recipe-test.js

```diff
@@ -204,21 +210,82 @@ describe('recipe', () => {
 });
 
 describe('filterRecipes', () => {
+  let expectedRecipes;
+  let filteredRecipes;
+  let nameSearched;
+  let ingredientSearched;
```

Comment:
totally non-blocking, but we need to be careful using shared variables across tests - don't think it matters much here, but it could lead to complication with more complex data manipulation.  I would generally recommend declaring these in a `beforeEach` hook.